### PR TITLE
Add deprecation notices

### PR DESCRIPTION
**Pull request by djbender**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # npm-login
 
+**Please note: This action is deprecated and should no longer be used. Please, migrate your workflows to the [actions/setup-node](https://github.com/actions/setup-node) action.**
+
```
Comment (rexfleischer) [inline README.md:4]: can we show the example you had in the other ticket? I would like people to just have to copy/paste to migrate.

File: action.yml [hunk 2]
```diff
@@ -20,6 +20,11 @@ inputs:
 runs:
   using: composite
   steps:
+    - shell: bash
+      run: |
+        echo "::notice::NOTE: This action is deprecated and is no longer maintained."
+        echo "::notice::Please, migrate to https://github.com/actions/setup-node"
+
```
Comment (rexfleischer) [inline action.yml:27]: good idea here!

Comment (djbender) [reply action.yml:27]: yeah!! it'll print messages in the Actions overview which is perfect. https://github.com/get-bridge/npm-login/actions/runs/1270482220

File: README.md [hunk 3]
```diff
@@ -3,6 +3,22 @@
 ![ci](https://github.com/get-bridge/npm-login/workflows/ci/badge.svg)
 ![tag](https://img.shields.io/github/v/tag/get-bridge/npm-login?sort=semver)
 
+**Please note: This action is deprecated and should no longer be used. Please, migrate your workflows to the [actions/setup-node](https://github.com/actions/setup-node) action. To authenticate with `actions/setup-node` use the following general approach which has been adapted from the official [documentation](https://github.com/actions/setup-node/blob/270253e841af726300e85d718a5f606959b2903c/docs/advanced-usage.md#use-private-packages):**
+
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        registry-url: http://npm.pkg.github.com
```
Comment (rexfleischer) [inline README.md:12]: there's no scope?

Comment (djbender) [reply README.md:12]: It defaults to the repo owner! So any action ran in the `@get-bridge` org will default to `scope: '@get-bridge'`.

**Review by rexfleischer — APPROVED** [review]
Looks good to me. If there's not a scope, then this is good.

**Pull request merged**